python/lsst/sims/utils/CodeUtilities.py

```diff
@@ -2,6 +2,38 @@
 import numpy as np
 import numbers
 
+def sims_clean_up():
+    """
+    This method will clean up data caches created by the sims software stack.
+    Any time a cache is added to the sims software stack, it can be added to
+    the list sims_clean_up.targets.  When sims_clean_up() is called, it will
+    loop through the contents of sims_clean_up.targets.  It will call pop()
+    on all of the contents of each sims_clean_up.target, and then reset each
+    sims_clean_up.target to either a blank dict or list (depending on what
+    the target was).
```

Comment:
Why do you return the empty targets? If you run more code you'll just be appending more lists and dicts to sims_clean_up.targets and still keeping empty lists and dicts around.